### PR TITLE
Create uno.md

### PR DESCRIPTION
**Pull request by MKFeuer**
## Type of PR

- [ X ] Translation update / creation
- [ ] Original file content (not typo) update
- [ ] Original file creation
- [ ] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Uno
+
+Lasst eure User gegeneinander Uno spielen!
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:3]:
```suggestion
Lasse deine Nutzer gegeneinander Uno spielen!
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+<ModuleOverview moduleName="uno" />
+
+## Funktionen {#features}
+Uno ist ein klassisches Kartenspiel - jetzt auch für deinen Eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schauen dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]:
```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Jeder Spieler beginnt mit sieben Karten. Die anderen Spieler können nur die Anzahl der Karten jedes Spielers sehen. Wenn ein Spieler keine Karte spielen kann, muss er eine Karte ziehen. Wenn ein Spieler keine Karten mehr hat, gewinnt er das Spiel.
+
+## Einrichtung {#setup}
+Die Einrichtung ist wirklich einfach – aktivieren sie einfach das Modul in [deinem SCNX-Dashboard](https://scnx.app/de/glink?page=bot/modules?query=uno&ref=scnx-app-docs) und schon kann es losgehen!
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:13]:
```suggestion
Die Einrichtung ist wirklich einfach - aktiviere einfach das Modul in [deinem SCNX-Dashboard](https://scnx.app/de/glink?page=bot/modules?query=uno&ref=scnx-app-docs) und schon kann es losgehen!
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Verwendung {#usage}
+
+Um ein Uno-Spiel zu starten, führen sie den Slash-Befehl `/uno` aus – andere Spieler können dann über die Schaltfläche „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf die Schaltfläche „Spiel starten“ geklickt hat, beginnt das Spiel.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:17]:
```suggestion
Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+:::tip Spiel Thread 
+Es ist am besten, einen Thread für Spieldiskussionen zu erstellen, damit der Kanal nicht zu voll mit spielbezogenen Nachrichten wird.
+:::
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:21]:
```suggestion
:::tip Spiel-Thread 
Es ist am besten, einen Thread für Spieldiskussionen zu erstellen, damit der Kanal nicht zu voll mit spielbezogenen Nachrichten wird.
:::
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+Inaktive Spieler werden nach 60 Sekunden, wenn sie an der Reihe sind, verwarnt und nach weiteren 60 Sekunden aus dem Spiel geworfen.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:23]:
```suggestion
Inaktive Spieler werden 60 Sekunden, nachdem sie an die Reihe gekommen sind verwarnt und nach weiteren 60 Sekunden aus dem Spiel entfernt.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Das Spiel selbst hat keine zeitliche Begrenzung.
+
+### Wie man spielt {#how-to-play}
+Spieler können nur Karten spielen, die dem aktuellen Wert oder der aktuellen Farbe der zuletzt gespielten Karte entsprechen. Ausnahmen gelten für die Sonderkarten „Farbwahl“.
+
+In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, und dann ist der nächste Spieler an der Reihe – beachten Sie, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:30]:
```suggestion
In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+### Karten und Specialeffekte {#cards}
+Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die Funktion „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Reverse“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:33]:
```suggestion
Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Reverse“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+Außerdem gibt es noch die Karten „Ziehe 2“ und „Farbwahl und ziehe 4“: Dadurch wird der nächste Spieler gezwungen, zwei oder vier Karten zu ziehen, sofern er nicht auch eine dieser Karten in seinem Kartenstapel hat. Hat er eine solche Karte, kann er damit die Anzahl der zu ziehenden Karten erhöhen und an den nächsten Spieler weitergeben, wo sich der ganze Vorgang wiederholt.
+
+## Commands {#commands}
+
+<SlashCommandExplanation />
+
+| Command | Description                                                                                                            |
+|---------|------------------------------------------------------------------------------------------------------------------------|
+| `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise finden sie im [Abschnitt zur Verwendung](#usage) oben.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:43]:
```suggestion
| Befehl | Beschreibung                                                                                                            |
|---------|------------------------------------------------------------------------------------------------------------------------|
| `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise findest du im [Abschnitt zur Verwendung](#usage) oben. 
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+|---------|------------------------------------------------------------------------------------------------------------------------|
+| `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise finden sie im [Abschnitt zur Verwendung](#usage) oben. 
+                                                                                                                                   |
+
+
+## Fehlerbehebung {#troubleshooting}
+
+<details>
+    <summary>Wie kann ich eine Karte spielen/Wo ist mein Kartendeck?</summary>
+    <li>Sie können eine neue Nachricht mit Ihrem aktuellen Kartendeck erhalten, einschließlich einer Schaltfläche zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klicken.</li>
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:51]:
```suggestion
    <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck erhalten, einschließlich einem Knopf zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klickst.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+</details>
+<details>
+    <summary>TDer Bot sagt: "⚠️️ Sie müssen die Uno!-Taste verwenden, bevor Sie Ihre vorletzte Karte verwenden!" - aber warum?</summary>
+    <li>Sie müssen zuerst auf Uno! klicken. Klicken Sie auf die Schaltfläche auf der ursprünglichen Spielnachricht, bevor Sie Ihre vorletzte Karte ausspielen. Dies gilt nicht für das Ausspielen der letzten Karte.</li>
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:55]:
```suggestion
    <summary>Der Bot sagt: "⚠️️️ Du musst den Uno!-Button nutzen, bevor du die vorletzte Karte legst! " - wieso?</summary>
    <li>Du musst zuerst auf "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>
```

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 1 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:3]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 2 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 3 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:13]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 4 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:17]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 5 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:21]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 6 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:23]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 7 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:30]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 8 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:33]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 9 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:43]: sorry hatte ich übersehen 

aber done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 10 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:51]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 11 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:55]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 1 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: No?

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
+# Uno
+
+Lasse deine User gegeneinander Uno spielen!
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:3]:
```suggestion
Lasse deine Nutzer gegeneinander Uno spielen!
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+<ModuleOverview moduleName="uno" />
+
+## Funktionen {#features}
+Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]:
```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchtest, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+:::
+
+Inaktive Spieler werden nach 60 Sekunden, nachdem sie an die Reihe gekommen sind, verwarnt und nach weiteren 60 Sekunden aus dem Spiel entfernt.
+
+Das Spiel selbst hat keine zeitliche Begrenzung.
+
+### Wie man spielt {#how-to-play}
+Spieler können nur Karten spielen, die dem aktuellen Wert oder der aktuellen Farbe der zuletzt gespielten Karte entsprechen. Ausnahmen gelten für die Sonderkarten „Farbwahl“.
+
+In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachten Sie, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:30]:
```suggestion
In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+
+### Karten und Specialeffekte {#cards}
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:32]:
```suggestion
### Karten und Spezialeffekte {#cards}
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Reverse“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.
+
+Außerdem gibt es noch die Karten „Ziehe 2“ und „Farbwahl und ziehe 4“: Dadurch wird der nächste Spieler gezwungen, zwei oder vier Karten zu ziehen, sofern er nicht auch eine dieser Karten in seinem Kartenstapel hat. Hat er eine solche Karte, kann er damit die Anzahl der zu ziehenden Karten erhöhen und an den nächsten Spieler weitergeben, wo sich der ganze Vorgang wiederholt.
+
+## Commands {#commands}
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:37]:
```suggestion
## Befehle {#commands}
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+| Befehl  | Beschreibung                                                                                                            |
+|---------|------------------------------------------------------------------------------------------------------------------------|
+| `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise findest du im [Abschnitt zur Verwendung](#usage) oben. 
+                                                                                                                                   |
+
+
+## Fehlerbehebung {#troubleshooting}
+
+<details>
+    <summary>Wie kann ich eine Karte spielen/Wo ist mein Kartendeck?</summary>
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:50]:
```suggestion
    <summary>Wie kann ich eine Karte spielen / Wo ist mein Kartendeck?</summary>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+    <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck erhalten, einschließlich einer Schaltfläche zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klickst.</li>
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:51]:
```suggestion
    <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck und einem Knopf zum Aktualisieren erhalten, indem du auf den Knopf „Eigene Karten ansehen“ in der ursprünglichen Spielnachricht klickst.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+</details>
+<details>
+    <summary>Der Bot sagt: "⚠️️ Du musst die Uno!-Button nutzen, bevor du die vorletzte Karte legst!" - aber wieso?</summary>
+    <li>u musst zuerst auf "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:55]:
```suggestion
    <summary>Der Bot sagt: "⚠️️️ Du musst den Uno!-Button nutzen, bevor du die vorletzte Karte legst!" - wieso?</summary>
    <li>Du musst zuerst auf den "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>
```

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 1 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: now jet

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 2 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no isue

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 3 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no issue

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 4 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no issue

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 5 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no issue

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 6 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no issue

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 12 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: no issue in my code

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 13 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 14 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:30]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 16 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:37]: sorry i miss it

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 15 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:32]: correckted

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 17 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:50]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 18 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:51]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 19 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:55]: i don't know wath happend during my las correction, but i fixed it now

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 20]
```diff
@@ -0,0 +1,56 @@
+# Uno
+
+Lasse deine Nutzer gegeneinander Uno spielen!
+
+<ModuleOverview moduleName="uno" />
+
+## Funktionen {#features}
+Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]:
```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchtest, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+
+Jeder Spieler beginnt mit sieben Karten. Die anderen Spieler können nur die Anzahl der Karten jedes Spielers sehen. Wenn ein Spieler keine Karte spielen kann, muss er eine Karte ziehen. Wenn ein Spieler keine Karten mehr hat, gewinnt er das Spiel.
+
+## Einrichtung {#setup}
+Die Einrichtung ist wirklich einfach – aktiviere einfach das Modul in [deinem SCNX-Dashboard](https://scnx.app/de/glink?page=bot/modules?query=uno&ref=scnx-app-docs) und schon kann es losgehen!
+
+## Verwendung {#usage}
+
+Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:17]:
```suggestion
Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausgeführt hat, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.
```

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 20 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md:8]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md (hunk 21 above)
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: solved

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+:::
+
+Inaktive Spieler werden nach 60 Sekunden, nachdem sie an die Reihe gekommen sind, verwarnt und nach weiteren 60 Sekunden aus dem Spiel entfernt.
+
+Das Spiel selbst hat keine zeitliche Begrenzung.
+
+### Wie man spielt {#how-to-play}
+Spieler können nur Karten spielen, die dem aktuellen Wert oder der aktuellen Farbe der zuletzt gespielten Karte entsprechen. Ausnahmen gelten für die Sonderkarten „Farbwahl“.
+
+In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```
[the remark this answers is not in the document]
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: done

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+
+### Karten und Spezialeffekte {#cards}
+Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Umkehren“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.
```
[the remark this answers is not in the document]
Comment (MKFeuer) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md]: done

**Review by Fridolin012 — APPROVED** [review]

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open-Source Developer Program. You can find details in your SCNX organization payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.

**Pull request merged**